testing/simapp/app.go

```diff
@@ -748,7 +749,7 @@ func (app *SimApp) GetBaseApp() *baseapp.BaseApp {
 }
 
 // GetStakingKeeper implements the TestingApp interface.
```

Comment:
update godoc

Reply:
> LGTM. Could you add a changelog entry?
> 
> I'm also okay leaving the `GetStakingKeeper` name as is and doing in a followup pr if it is still desired. I don't have strong preferences though

then it's no longer API breaking, correct?